app/src/main/java/com/khaithu/a3ctech_android_final_project/presenter/SearchPresenter.kt

```diff
@@ -0,0 +1,67 @@
+package com.khaithu.a3ctech_android_final_project.presenter
+
+import android.util.Log
+import com.khaithu.a3ctech_android_final_project.BuildConfig
+import com.khaithu.a3ctech_android_final_project.api.RestClient
+import com.khaithu.a3ctech_android_final_project.helper.Constant
+import com.khaithu.a3ctech_android_final_project.model.ResultMovie
+import com.khaithu.a3ctech_android_final_project.model.Search
+import com.khaithu.a3ctech_android_final_project.model.enum.LanguageEnum
+import com.khaithu.a3ctech_android_final_project.presenter.interfacePre.ISearchPresenter
+import com.khaithu.a3ctech_android_final_project.service.IMovieService
+import com.khaithu.a3ctech_android_final_project.view.interfaceView.ISearchView
+import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers
+import io.reactivex.rxjava3.core.Observable
+import io.reactivex.rxjava3.core.Observer
+import io.reactivex.rxjava3.disposables.CompositeDisposable
+import io.reactivex.rxjava3.disposables.Disposable
+import io.reactivex.rxjava3.schedulers.Schedulers
+
+class SearchPresenter(val mISearchView: ISearchView) : BasePresenter(), ISearchPresenter {
```

Comment:
```suggestion
class SearchPresenter(var mView: ISearchView?) : BasePresenter(), ISearchPresenter {
```
assign `mView = null` when destroy presenter